Handbook/Resources.md

```diff
@@ -81,14 +81,14 @@ Useful materials that give an overview of Cook County's property tax system and
 
 ## Git, Linux, and DevOps
 
-- [:exclamation: The Missing Semester of Your CS Education](https://missing.csail.mit.edu) - Start here if you're unfamiliar with the command line, build tooling, scripting, etc.
+- [:exclamation: The Missing Semester of Your CS Education](https://missing.csail.mit.edu) - Start here if you're unfamiliar with the CLI, build tooling, scripting, etc.
```

Comment:
**[Nitpick, non-blocking]** Hmm, I'm not used to seeing "CLI" used with a definitive article; I'm more accustomed to seeing "a CLI" in reference to any one of many tools that process I/O via a terminal (see e.g. [the Wikipedia page](https://en.wikipedia.org/wiki/Command-line_interface) for some more sample usage). I also think that acronyms are generally best avoided in contexts where the reader isn't guaranteed to understand what they mean. Is there a particular reason to prefer "the CLI" here over the previous "the command line" (or even "command line interfaces")?

Reply:
I shortened it because the line broke and looked funny 😆 , I'll revert it.